test/functional/mempool_ephemeral_anchor.py

```diff
@@ -331,11 +331,12 @@ def test_xor_rbf(self):
         self.generate(node, 1)
         self.assert_mempool_contents(expected=[second_package_txns[1]], unexpected=[])
 
-        # Parent can still be prioritised
+        # Parent can't be prioritised; RPC call succeeds but is modified fees are ignored if it has EAs
```

Comment:
"i***t***s modified fees" ?

Reply:
removed the test since it's duplicated by v3 work which now trims lower than min relay fee txns